go.mod

```diff
@@ -2,11 +2,15 @@ module github.com/atlassian/terraform-provider-artifactory
 
 require (
 	github.com/atlassian/go-artifactory/v2 v2.5.0
+	github.com/frankban/quicktest v1.7.2 // indirect
 	github.com/go-delve/delve v1.6.0 // indirect
 	github.com/hashicorp/terraform-plugin-sdk v1.7.0
-	github.com/jfrog/jfrog-client-go v0.13.1
-	github.com/stretchr/testify v1.5.1
 	github.com/jasonwbarnett/go-xray v0.1.4
```

Comment:
`jasonbarnet/go-xray` should no longer be needed

Reply:
The `artifactory_xray_policy` resource still uses `go-xray`.